index.d.ts

```diff
@@ -0,0 +1,14 @@
+import * as Handler from './types/handler';
+import * as Options from './types/options';
```

Comment:
To be honest, having multiple `.d.ts` files feel like overkill to me, considering that our typings are less than 30 lines of code in total. Not a big deal though, if you prefer this way then I am ok with that.